pages/15.Reports/docs.en.md

```diff
@@ -16,24 +16,41 @@ Highly customizable reports can be generated through Mautic's Report menu.
 ## Data Sources
 
 Choose the data source appropriate to the report you want. Each data source has a different set of available columns, filters and graphs.
- 
-![](data-source.png)
+
+![The Details tab in the Edit Report window](data-source.png)
 
 ## Configuration
 
 Each report can be customized to include the columns of choice. Filter data based on set criteria and/or set a specific order for the data.
 In addition you can also group by and select different function operators to calculate fields. Note that when you select functions operators a totals row will be added to the report. This totals row will not be exported when selecting to export a report.
- 
-![](config.png)
+
+![The Data tab in the Edit Report window](config.png)
 
 ## Graphs
 
 Some reports have graphs available. Select the graph desired from the left list - it will move to the right and will be part of the report.
 
-![](graphs.png)
+![The Graphs tab in the Edit Report window](graphs.png)
 
 ## Dashboard Widget
 
-Each graph of each report is made available as a widget on the dashboard allowing complete customization of the dashboard. 
- 
-![](widget.png)
\ No newline at end of file
+Each graph of each report is made available as a widget on the dashboard allowing complete customization of the dashboard.
+
+![The Add Widget window on the Dashboard](widget.png)
+
+## Schedule
+
+Enable or disable sending reports via email by using the toggle switch.
+
+You can schedule emails to send reports to one or more email addresses. In the To field, enter a comma-separated list of email addresses and set the frequency of sending reports by choosing day, week, or month from the drop-down list.
+
+![The Schedule tab in the Edit Report window](schedule.png)
+
+To be able to send scheduled reports, the following cron command is required:
+
+```
+php /path/to/mautic/app/console mautic:reports:scheduler [--report=ID]
+```
+The `--report=ID` argument allows you to specify a report by ID if required. For more information, see [Cron jobs][cron-jobs].
+
+[cron-jobs]:</setup/cron-jobs>
```

Comment:
We need a space here between the : and the < (see https://www.markdownguide.org/basic-syntax/#formatting-the-second-part-of-the-link for reference).

Also, it is kinder to the reader if we send them direct to the appropriate part of the page using the heading anchors.

```suggestion
[cron-jobs]: </setup/cron-jobs#send-scheduled-reports>
```

Reply:
Noted these points about using macro links. I've updated the doc with your suggestion @RCheesley.